app/scripts/configs/tracks-info.js

```diff
@@ -1157,7 +1157,7 @@ export const TRACKS_INFO = [
     type: 'horizontal-gene-annotations',
     datatype: ['gene-annotation'],
     local: false,
-    defaultHeight: 55,
+    defaultHeight: 90,
```

Comment:
Should `defaultWidth` for `vertical-gene-annotations` reflect this change?

Reply:
Yes it should indeed. Nice catch!